.github/workflows/ruby.yml

```diff
@@ -1,6 +1,6 @@
 name: Ruby
 
-on: [push, pull_request]
+on: [push]
```

Comment:
Removing the `pull_request` event to avoid having two lines of CI for the same status (in a different commit).